### ISSUE TITLE
Differences between jellejurre VPM support

### ISSUE THREAD

**Issue by lyuma** [anatawa12/Av3Emulator#2]
Hi anatawa12,
Thanks so much for taking this on!

It looks like you and jellejurre were coincidentally developing VPM at the same time.
https://github.com/lyuma/Av3Emulator/pull/81

I do like how your implementation handles the GestureManager assembly reference. There should be no need for a special script to detect the library.

Anyway just wanted to let you know in case you weren't aware. maybe the two can be reconciled somehow

**Comment by anatawa12** [comment]
Thank you for letting me know!

I'm sorry to say that, I thought Av3Emu is not being maintained, I fork this repo to release av3emu in my vpm repository.

I found jellejurre's fork in pullreqs in your repo but I think their detecting GestureManager is not a good implementation so I didn't use that implementation.

If the origin repo is going to be maintained, I back to contributing origin.

**Comment by lyuma** [comment]
Hi, @anatawa12.

I have talked with @jellejurre and @Dreadrith and consensus is that we will move forward with developing Av3Emulator under lyuma. I have invited all of you as collaborators.

jellejurre also helped merge in many of the improvements you made to the VPM script, including detecting the GestureManager package.

I believe we have a plan to move forward with making lyuma/Av3Emulator use VPM officially. You are welcome to join in the development there if you want. It is also fine to use a fork or submit pull requests.

The decision to use VPM was a big roadblock to me. Now that we are resolving this, I hope it will be easier to develop Av3Emulator in the future. Thanks so much for your contributions so far.

I will close this issue since I only used it for discussion, not an actual bug.
-Lyuma
